src/include/txn.i

```diff
@@ -542,30 +547,64 @@ __wt_txn_visible(
  * __wt_txn_upd_visible --
  *	Can the current transaction see the given update.
  */
-static inline bool
+static inline WT_VISIBLE_TYPE
 __wt_txn_upd_visible(WT_SESSION_IMPL *session, WT_UPDATE *upd)
```

Comment:
There are callers of this function that don't seem to have changed - (`rec_write.c` and `bt_ret.c`). That surprises me.